dotnet-efcore/EntityFrameworkCoreVsDapper/EntityFrameworkCoreVsDapper/Repositories/PersonsRepositoryDapper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using BenchmarkDotNet.Attributes;
+using Dapper;
+using EntityFrameworkCoreVsDapper.Models;
+using System.Data.SqlClient;
+
+namespace EntityFrameworkCoreVsDapper.Repositories
+{
+    public class PersonsRepositoryDapper : IPersonsRepository
+    {
+        //ADD THE CONNECTION STRING TO YOUR DATABASE HERE
+        private string ConnectionString = "";
+
+        [Benchmark]
+        public List<Person> Get_1000_Persons()
```

Comment:
Instead of copy/pasted code, this would be better with a private method that takes a count parameter and then each of these methods internally could call it. You could even change the interface to be a single method with a count parameter, and then use a [parameterized benchmark](https://andreyakinshin.github.io/bdn-docs-wip/articles/features/parameterization.html#sample-introarguments) to run the three versions. Either way is probably okay, but for sure the internal duplicate code should be extracted to a private method. The same concept applies for the EFCore code.